src-docs/src/views/theme/computed.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+import { tint, shade } from '../../../../src/services/theme/theme';
+import { EuiIcon } from '../../../../src/components/icon';
+import { EuiCode } from '../../../../src/components/code';
+import {
+  computed,
+  EuiThemeProvider,
+  useEuiTheme,
+} from '../../../../src/services';
+
+const Box: FunctionComponent<{ children: ReactNode }> = ({ children }) => {
+  const [theme] = useEuiTheme();
+
+  return (
+    <div
+      css={{
+        background: theme.colors.euiColorHighlight,
+        padding: theme.sizes.euiSizeXL,
+        color: theme.colors.euiColorPrimaryText,
+      }}>
+      <p>
+        <EuiIcon type="stopFilled" color={theme.colors.euiColorPrimary} />{' '}
+        {children}
+      </p>
+    </div>
+  );
+};
+
+export default () => {
+  const primaryOverrides = {
+    colors: {
+      light: {
+        euiColorPrimary: '#db1dde',
+        euiColorHighlight: computed(
+          ['colors.euiColorPrimary'],
+          ([euiColorPrimary]) => tint(euiColorPrimary, 0.8)
+        ),
+        // Hmm, can't add a custom key?
```

Comment:
Not possible to add a new key through `overrides`.
A couple weeks ago I would have said its not possible because of technical limitations, but now I'm not sure that that's strictly true. It would certainly add complexity and have restrictions (because of value computing), but it's something we could discuss (and influence changing the prop name to something other than `overrides`).

Right now, the only way to "extend" a theme is to create a new one that has no relationship to parent themes, even if they overlap. For example:

```js
const newTheme = buildTheme(
  {
    ...euiThemeDefault,
    custom: '#000',
  },
  'CUSTOM'
);
```
So even if you used `overrides` to set `euiColorPrimary` in a parent theme this `newTheme` would be unaffected.